src/chains/migrations/0038_auto_20221024_1543.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.0.5 on 2022-10-24 15:43
+
+from django.apps.registry import Apps
+from django.contrib.auth.models import Group, Permission
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+def create_support_group(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    support_group, created = Group.objects.get_or_create(name="support")
+    permission = Permission.objects.filter(codename="change_chain")
+    if permission:
+        support_group.permissions.add(permission[0])
```

Comment:
If you only expect to get one permission you can do `Permission.objects.get(codename="change_chain")`